hub/tests/test_views.py

```diff
@@ -59,6 +65,11 @@ def test_explore_datasets_json_page(self):
         response = self.client.get(url)
         self.assertEqual(response.status_code, 200)
 
+    def test_explore_view_with_many_to_one(self):
+        url = f"{reverse('explore_csv')}?mp_appg_membership__exact=MadeUpAPPG"
+        response = self.client.get(url)
+        self.assertEqual(response.status_code, 200)
+
```

Comment:
might be worth checking that the data returned looks ok too?

Reply:
Good shout! That's been included